Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM ubuntu:22.04
 
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     python3.11 \
+    python3.11-dev \
```

Comment:
we need this for Whisper to work properly, at least with word_timestamps turned on